### PR TITLE
Add alert-bar of IE

### PR DESCRIPTION
**Pull request by hh1680651**

### CODE REVIEW

File: twittermap/public/javascripts/timeseries/controllers.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
                 .text('Reset')
                 .attr('href',"#")
                 .on("click", function() { timeSeries.filterAll(); dc.redrawAll(); requestFunc(minDate, maxDate);})
-                .style("position", "absolute")
+//                .style("position", "absolute")
```
Comment (JavierJia) [inline twittermap/public/javascripts/timeseries/controllers.js]: reason to comment out this line?

File: twittermap/public/javascripts/map/controllers.js [hunk 2]
```diff
@@ -1,5 +1,27 @@
 angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
   .controller('MapCtrl', function($scope, $window, $http, $compile, cloudberry, leafletData, cloudberryConfig) {
+
+    // add alert of IE
+    if (!L.Browser.ie) {
+    ieAlert();
+    }
+
+    function ieAlert() {
+    var alertDiv = document.getElementsByTagName("alert-bar")[0];
```
Comment (JavierJia) [inline twittermap/public/javascripts/map/controllers.js]: add indentation?

File: twittermap/public/javascripts/map/controllers.js [hunk 3]
```diff
@@ -1,5 +1,27 @@
 angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
   .controller('MapCtrl', function($scope, $window, $http, $compile, cloudberry, leafletData, cloudberryConfig) {
+
+    // add alert of IE
+    if (!L.Browser.ie) {
+    ieAlert();
```
Comment (JavierJia) [inline twittermap/public/javascripts/map/controllers.js]: since it is a special function, can we just put the implementation here? the `alert` shouldn't be used anywhere else?

**Comment by chenlica** [comment]
The yellow message looks good to me!

File: twittermap/public/javascripts/map/controllers.js [hunk 4]
```diff
@@ -1,25 +1,23 @@
+// add alert of IE
```
Comment (JavierJia) [inline twittermap/public/javascripts/map/controllers.js]: wrong comment?

File: twittermap/app/views/twittermap/main.scala.html [hunk 5]
```diff
@@ -3,6 +3,7 @@
 @import play.api.libs.json.Json
 
 <!DOCTYPE html>
+
```
Comment (JavierJia) [inline twittermap/app/views/twittermap/main.scala.html]: we should avoid unnecessary changes.

File: twittermap/public/javascripts/timeseries/controllers.js (hunk 1 above)
Comment (JavierJia) [inline twittermap/public/javascripts/timeseries/controllers.js]: any answer for this comment?

File: twittermap/app/views/twittermap/main.scala.html (hunk 5 above)
Comment (waans11) [inline twittermap/app/views/twittermap/main.scala.html]: Agreed.

File: twittermap/app/views/twittermap/index.scala.html [hunk 6]
```diff
@@ -1,15 +1,15 @@
 
 @(title: String, ws: String, startDate: String, sentimentEnabled: Boolean, sentimentUDF: String, removeSearchBar: Boolean, predefinedKeywords: Seq[String], isDrugMap: Boolean) @main(title, ws, startDate, sentimentEnabled, sentimentUDF, removeSearchBar, predefinedKeywords, isDrugMap){
 <div xmlns="http://www.w3.org/1999/html" ng-controller="AppCtrl">
-
```
Comment (waans11) [inline twittermap/app/views/twittermap/index.scala.html]: This line change might not be necessary.

File: twittermap/app/views/twittermap/index.scala.html [hunk 7]
```diff
@@ -1,15 +1,15 @@
...
   <div class="map-group">
+    <alert-bar></alert-bar>
 
 
     <map lat="39.5" lng="-96.35" zoom="4"></map>
 
     <div id="logo">
     </div>
-    
+
```
Comment (waans11) [inline twittermap/app/views/twittermap/index.scala.html]: This format change, too. Let's make changes whenever you touch that area.

File: twittermap/app/views/twittermap/index.scala.html [hunk 8]
```diff
@@ -18,8 +18,8 @@
       </div>
    </div>
 
-    
-   </div>
+
+  </div>
```
Comment (waans11) [inline twittermap/app/views/twittermap/index.scala.html]: Same.

File: twittermap/app/views/twittermap/index.scala.html [hunk 9]
```diff
@@ -29,7 +29,6 @@
   <div class="stats">
     <time-series id="chart"></time-series>
   </div>
-
```
Comment (waans11) [inline twittermap/app/views/twittermap/index.scala.html]: Same.

File: twittermap/public/javascripts/map/controllers.js [hunk 10]
```diff
@@ -130,6 +150,7 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
 
 
     function setInfoControl() {
+
```
Comment (waans11) [inline twittermap/public/javascripts/map/controllers.js]: Unnecessary format change

File: twittermap/public/stylesheets/main.css [hunk 11]
```diff
@@ -18,7 +18,7 @@ body {
   bottom: 0;
   left: 96%;
   width: 24%;
-  height: 700px;
+  height: 90%;
```
Comment (waans11) [inline twittermap/public/stylesheets/main.css]: @JavierJia : Is this OK?

Comment (JavierJia) [inline twittermap/public/stylesheets/main.css]: I think it's better to be relative than absolute in general. But I don't know if there is any reason for choosing this particular `700px`. @hh1680651  you may want to change the resolution using develop mode to confirm that the new value `90%` won't cause much trouble?

File: twittermap/public/javascripts/timeseries/controllers.js (hunk 1 above)
Comment (hh1680651) [inline twittermap/public/javascripts/timeseries/controllers.js]: uncommented

File: twittermap/public/javascripts/map/controllers.js (hunk 3 above)
Comment (hh1680651) [inline twittermap/public/javascripts/map/controllers.js]: Done.

File: twittermap/public/javascripts/map/controllers.js (hunk 2 above)
Comment (hh1680651) [inline twittermap/public/javascripts/map/controllers.js]: Done.

File: twittermap/public/javascripts/map/controllers.js (hunk 4 above)
Comment (hh1680651) [inline twittermap/public/javascripts/map/controllers.js]: deleted

File: twittermap/app/views/twittermap/main.scala.html (hunk 5 above)
Comment (hh1680651) [inline twittermap/app/views/twittermap/main.scala.html]: removed

File: twittermap/app/views/twittermap/index.scala.html (hunk 6 above)
Comment (hh1680651) [inline twittermap/app/views/twittermap/index.scala.html]: removed

File: twittermap/app/views/twittermap/index.scala.html (hunk 7 above)
Comment (hh1680651) [inline twittermap/app/views/twittermap/index.scala.html]: Okay I'll keep that in mind.

File: twittermap/app/views/twittermap/index.scala.html (hunk 8 above)
Comment (hh1680651) [inline twittermap/app/views/twittermap/index.scala.html]: removed

File: twittermap/app/views/twittermap/index.scala.html (hunk 9 above)
Comment (hh1680651) [inline twittermap/app/views/twittermap/index.scala.html]: removed

File: twittermap/public/javascripts/map/controllers.js (hunk 10 above)
Comment (hh1680651) [inline twittermap/public/javascripts/map/controllers.js]: removed

File: twittermap/public/stylesheets/main.css (hunk 11 above)
Comment (hh1680651) [inline twittermap/public/stylesheets/main.css]: I changed this to suit alert-bar ver 1 (on the bottom) but for the new one it's unnecessary so I removed this change in new pull request.
